apps/allocations/app/components/App/App.js

```diff
@@ -110,59 +111,76 @@ class App extends React.Component {
     this.setState({ panel: { visible: false } })
   }
 
+  handleResolveLocalIdentity = address => {
+    return this.props.api.resolveAddressIdentity(address).toPromise()
+  }
+
+  handleShowLocalIdentityModal = address => {
+    return this.props.api
+      .requestAddressIdentityModification(address)
+      .toPromise()
+  }
+
   render() {
     const { panel } = this.state
     const { displayMenuButton = false } = this.props
     const PanelContent = panel.content
+
+    console.log(this.props)
+
     return (
       // TODO: Profile App with React.StrictMode, perf and why-did-you-update, apply memoization
       <Main>
-        <AppView
-          padding={0}
-          appBar={
-            <AppBar
-              endContent={
-                <AppTitleButton
-                  caption="New Account"
-                  onClick={this.newAccount}
-                />
-              }
-            >
-              <AppTitle title="Allocations" displayMenuButton={displayMenuButton} />
-            </AppBar>
-          }
-        >
-          <Accounts
-            accounts={
-              this.props.accounts !== undefined ? this.props.accounts : []
-            }
-            onNewAccount={this.newAccount}
-            onNewAllocation={this.newAllocation}
-            app={this.props.app}
-          />
-          <Payouts
-            payouts={
-              this.props.payouts !== undefined ? this.props.payouts : []
+        <IdentityProvider
+          onResolve={this.handleResolveLocalIdentity}
+          onShowLocalIdentityModal={this.handleShowLocalIdentityModal}>
+          <AppView
+            padding={0}
```

Comment:
This will probably conflict with #903 PR, not an issue right now, just something worth to take care when merging

Reply:
ok!